src/main/java/core/basesyntax/service/work/with/file/ReportImpl.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service.work.with.file;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.service.operation.Operation;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.OperationStrategy;
+import core.basesyntax.service.operation.OperationStrategyImpl;
+import core.basesyntax.service.operation.PurchaseOperationHandler;
+import core.basesyntax.service.operation.ReturnOperationHandler;
+import core.basesyntax.service.operation.SupplyOperationHandler;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReportImpl implements Report {
+    private static final String HEADER = "fruit,quantity";
+
+    @Override
+    public String writeReport(List<String> informationFromFile) {
+        FruitDao fruitDao = new FruitDaoImpl();
+        StringBuilder report = new StringBuilder();
+        report.append(HEADER).append(System.lineSeparator());
+        int counter = 0;
+        for (Map.Entry<Fruit, Integer> entry : fruitDao.getAllFruits()) {
+            report.append(fruitDao.getFruit(entry.getKey().getFruitName())
+                    .getFruitName()).append(",")
+                    .append(fruitDao.getAmount(entry.getKey().getFruitName()));
+            counter++;
+            if (counter < fruitDao.getSize()) {
+                report.append(System.lineSeparator());
+            }
+        }
+        return report.toString();
+    }
+
+    @Override
+    public void writeReportToFile(String report, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file", e);
+        }
+        Storage.fruits.clear();
```

Comment:
let's remove this, You can do this in your tests but not here. Imagine if your diploma project erased after printing